pkg/operator/workload_controller_openshiftapiserver_v311_00.go

```diff
@@ -172,6 +180,32 @@ func manageOpenShiftAPIServerClientCA_v311_00_to_latest(client coreclientv1.Core
 	return caChanged, nil
 }
 
+func manageOpenShiftAPIServerImageImportCA_v311_00_to_latest(openshiftConfigClient openshiftconfigclientv1.ConfigV1Interface, client coreclientv1.CoreV1Interface, recorder events.Recorder) (bool, error) {
+	const imageImportCA = "image-import-ca"
```

Comment:
move this to global and rename to `imageImportCAName`

Reply:
I will do this, but please note that i followed the pattern from 10 lines up:

```
        const apiserverClientCA = "client-ca"
 	_, caChanged, err := resourceapply.SyncConfigMap(client, recorder, kubeAPIServerNamespaceName, apiserverClientCA, targetNamespaceName, apiserverClientCA)
```